tests/integration/components/nrg-radio/component-test.js

```diff
@@ -1,35 +1,44 @@
-import { find, findAll, render, click } from '@ember/test-helpers';
+import { click, find, findAll, render } from '@ember/test-helpers';
 import { setupRenderingTest } from 'ember-qunit';
 import hbs from 'htmlbars-inline-precompile';
 import { module, test } from 'qunit';
 
-module('Integration | Component | nrg-radio', function(hooks) {
+module('Integration | Component | nrg-radio', function (hooks) {
   setupRenderingTest(hooks);
 
-  test('it becomes checked when its value is selected', async function(assert) {
-    await render(hbs`<NrgRadio @value="1" @selectedValue="1" />`);
+  test('it becomes checked when its value is selected', async function (assert) {
+    this.selectedValue = '1';
+    await render(
+      hbs`<NrgRadio @value="1" @model={{this}} @valuePath="selectedValue" />`
+    );
     assert.dom('.ui.checkbox.radio input').isChecked();
   });
 
-  test('label is clickable', async function(assert) {
-    await render(hbs`<NrgRadio @value="1" />`);
-    assert.dom('.ui.checkbox.radio input').isNotChecked();
-    await click('label');
-    assert.dom('.ui.checkbox.radio input').isChecked();
+  test('input is clickable', async function (assert) {
+    this.selectedValue = '0';
+    await render(
+      hbs`<NrgRadio @value="1" @model={{this}} @valuePath="selectedValue" />`
+    );
+    assert.equal(this.selectedValue, '0');
+    await click(find('.ui.checkbox.radio input'));
```

Comment:
You can pass a selector directly to the `click` function, eg:
```suggestion
    await click('.ui.checkbox.radio input');
```